tests/test_pipelinesBuildCancelTest.py

```diff
@@ -0,0 +1,31 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import unittest
+
+from azure_devtools.scenario_tests import AllowLargeResponse
+from .utilities.helper import DevopsScenarioTest, disable_telemetry, set_authentication, get_test_org_from_env_variable
+
+DEVOPS_CLI_TEST_ORGANIZATION = get_test_org_from_env_variable() or 'Https://dev.azure.com/dhilmathy'
+
+class PipelinesBuildCancelTests(DevopsScenarioTest):
+    @AllowLargeResponse(size_kb=3072)
+    @disable_telemetry
+    @set_authentication
+    def test_build_cancel(self):
+        self.cmd('az devops configure --defaults organization=' + DEVOPS_CLI_TEST_ORGANIZATION + ' project=buildtests')
+
+        build_definition_name = 'BuildTests Definition1'
```

Comment:
I think this will fail in PR build since the definition will not be existing in the test setup for us. 
Either use the existing definition in the test setup or create an E2E test creating the definition and then queueing.

Reply:
@atbagga I got the build definition name _"BuildTests Definition1"_ from existing [test_pipelinesBuildTagTest][1], [test_pipelinesBuildDefinitionsTest][2], [test_pipelinesBuildTest][3] tests. So, I think it will be present in test org as well. Please confirm.

 [1]: https://github.com/azure/azure-devops-cli-extension/blob/master/tests/test_pipelinesBuildTagTest.py#L18-L21
 [2]: https://github.com/dhilmathy/azure-devops-cli-extension/blob/BuildCancelCommand/tests/test_pipelinesBuildDefinitionsTest.py#L19-L21
 [3]: https://github.com/dhilmathy/azure-devops-cli-extension/blob/BuildCancelCommand/tests/test_pipelinesBuildTest.py#L18-L21

Reply:
Great you used the same! 
My bad I missed checking.